packages/client/src/world/controller.ts

```diff
@@ -195,6 +195,7 @@ export function getPhysicalInteractions(
   const interactions: Interactions[] = [];
   const item = physical as Item;
   const isOwner = item.isOwnedBy(currentCharacter?.community_id);
+  // console.log("Mob community:", currentCharacter?.community_id, "item ownedby:", item.ownedBy, "Is Owner:", isOwner);
```

Comment:
We should delete these instead of commenting it out. Since it adds noise to the codebase